cromwell_tools/cromwell_tools.py

```diff
@@ -153,7 +153,7 @@ def start_workflow(
 
     :return requests.Response response: HTTP response from cromwell.
     """
-    if validate_labels:
+    if validate_labels and label is not None:
```

Comment:
Why is this necessary? I would imagine if you set `validate_labels` to True, it indicates that you will pass in a label to the workflow right?

Reply:
validate_labels is `True` by default, and not set by the `cromwell-tools run` command line, therefore causes the cli to fail under the default specification.

Reply:
The defaults are `label=None` and `validate_labels=True`. Maybe the default for `validate_labels` should be `False` then, but I imagine you would more often want to validate that the labels are correct.

Reply:
OK, I see, sorry I missed the default value here with what we have in Lira. 

> validate_labels is `True` by default

It's defo a bug then. Nice catch!